panel/config.py

```diff
@@ -294,13 +300,50 @@ def __call__(self, *args, **params):
                 return
             with param.logging_level('ERROR'):
                 hv.plotting.Renderer.load_nb(config.inline)
-                nb_load = True
+                if hasattr(hv.plotting.Renderer, '_render_with_panel'):
+                    nb_load = True
 
         if not nb_load and hasattr(ip, 'kernel'):
             load_notebook(config.inline)
         panel_extension._loaded = True
 
 
+    def _apply_signatures(self):
+        from inspect import Parameter, Signature
+        from .viewable import Viewable
+
+        descendants = param.concrete_descendents(Viewable)
+        for cls in reversed(list(descendants.values())):
+            if cls.__doc__.startswith('params'):
```

Comment:
Is this to remove the docstring hint? If so I *believe* you can just disable this behavior at the param level with a flag somewhere (though it is possible that I am misremembering)

Reply:
>If so I believe you can just disable this behavior at the param level with a flag somewhere

There is a flag but it would have to be disabled on import.

Reply:
Right. You could set the flag as soon as param is imported.

Reply:
I think it's problematic, because if any other param based library gets imported in the process of importing panel then all those classes are affected too (don't think that does happen but I may very well be wrong).

Reply:
I don't think that would matter too much as those docstring hints are no longer supported by IPython and we'll be dropping Python 2 soon enough (and assuming Python versions that support ``__signature__``). Still just a suggestion and up to you!

Reply:
Will leave it for now.